packages/x-components/src/x-modules/search/components/promoteds-list.vue

```diff
@@ -6,11 +6,11 @@
         @binding {Vue | string} animation - Animation to animate the elements.
```

Comment:
change `injected search items to render.` to `injected list items to render.` in line 5